sandbox/src/services/patient-searcher.js

```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
                 resource: patient,
             })
         });
+        return response;
     } else {
-        delete response.entry
-    }
+        return {
+            resourceType: "OperationOutcome",
+            timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
+            "issue": [{"severity": "warning", "code": "not-supported", 
```

Comment:
Do the keys need to have quotation marks around them?

Reply:
converted to boom.notImplemented error.